docs/docs/guides/persistence.md

```diff
@@ -54,7 +54,7 @@ Here is a basic example:
 ```jsx
 const initializeState = ({set}) => {
   for(const [key, value] of Storage.entries()) {
-    set(getAtomWithKey(key), JSON.parse(value)).value;
```

Comment:
Given the example code above, it was serialized as an object with a `value` property.

Reply:
Got it!
```js
Storage.setItem(
  modifiedAtom,
  JSON.stringify({value: atomValues.get(modifiedAtom)}),
);
```
It should be `JSON.parse(value).value` rathar than `set().value`.